pkg/trace/api/container_linux_test.go

```diff
@@ -51,6 +51,7 @@ func TestConnContext(t *testing.T) {
 	if err := os.Chmod(sockPath, 0o722); err != nil {
 		t.Fatalf("error setting socket permissions: %v", err)
 	}
+	ln = NewMeasuredListener(ln, "uds_connections", 10)
```

Comment:
Does this addition make this test fail without you fixes in the other files?

Reply:
Yes it does, and that's exactly the purpose.